pkg/collect/copy.go

```diff
@@ -109,9 +115,59 @@ func copyFiles(c *Collector, client *kubernetes.Clientset, pod corev1.Pod, copyC
 		return nil, errors
 	}
 
-	return map[string][]byte{
-		copyCollector.ContainerPath: output.Bytes(),
-	}, nil
+	for _, file := range filepaths {
+		if file != "" {
+			command := []string{"cat", file}
+
+			req := client.CoreV1().RESTClient().Post().Resource("pods").Name(pod.Name).Namespace(pod.Namespace).SubResource("exec")
```

Comment:
This is unnecessarily verbose.

```
req := client.Corev1().Pods(pod.Namespace).Get(pod.Name)
```

Is cleaner and how we use the clientset

Reply:
@CodeLingoBot capture unnecessary use of client-go restClient

Reply:
Hi Marc! I didn't have much contact with the client set yet, so I left that line as it was. The way you suggest returns another type of variable and requires another type of variable too instead of pod.Name. I didn't investigate it further because I am sure I will have to deal with the clientset in later PR's, and I didn't wanted to delay the copy collector. 
That said I can either change it later, once I've dealt with the client set, or look into it now, as you prefer!